.globalconfig

```diff
@@ -10,7 +10,8 @@ dotnet_diagnostic.CA1001.severity = silent
 dotnet_diagnostic.CA1002.severity = none
 
 # CA1003: Use generic event handler instances
-dotnet_diagnostic.CA1003.severity = none
+dotnet_diagnostic.CA1003.severity = warning
+dotnet_code_quality.ca1003.api_surface = private, internal
```

Comment:
Please move this in GlobalConfig for consistency.

Reply:
move to where?

Reply:
https://github.com/PowerShell/PowerShell/blob/72608fb829881727f7d9b6295fb133613a30b6fe/.globalconfig#L12-L13

Reply:
This is already in `.globalconfig`?